helix-term/src/application.rs

```diff
@@ -406,6 +406,17 @@ impl Application {
 
         self.event_loop().await;
 
+        tokio::time::timeout(
+            Duration::from_millis(500),
+            future::join_all(
+                self.editor
+                    .language_servers
+                    .iter_clients()
```

Comment:
Rather than exposing `iter_clients`, could all of this including `join_all` just be a function on the editor?

Reply:
That seems like a good idea, I'd have to add a `futures_util` dependency to `helix-view` though

Reply:
We still need the `iter_clients` because `inner` field of `Registry` is private so there is no other way I think to access the clients from `Editor`